CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+1.9.0-RC1-velero_1.0.0 / 2020-04-07
+========================
+* support for local snapshot ([#53](https://github.com/openebs/velero-plugin/pull/53),[@mynktl](https://github.com/mynktl))
```

Comment:
should this be local snapshot and restore? and also does this support snapshot and restore volume into another namespace?

Reply:
yes, updated the comment. Thanks!